Source/HTTP/Android/http_android.cpp

```diff
@@ -3,13 +3,77 @@
 #include "pch.h"
 #if ANDROID_API
 #include <httpClient/httpClient.h>
+#include "android_http_request.h"
+
+void Internal_HCHttpCallInitialize(void* context)
+{
+    HttpRequest::InitializeJavaEnvironment(reinterpret_cast<JavaVM*>(context));
+}
 
 void Internal_HCHttpCallPerform(
```

Comment:
>Internal_HCHttpCallPerform [](start = 5, length = 26)

I think we want to associate a context with the perform function, so that we don't need to use statics for the JavaVM and associated pointers